lnrpc/rpc.proto

```diff
@@ -793,6 +793,21 @@ service Lightning {
             body: "*"
         };
     };
+
+    /** lncli: `channelrevenue`
+    ChannelRevenue returns channel revenue over the period requested.
+    If a zero start time is provided, revenue is calcualted from channel
+    open. If a zero end time is provided, revenue is calculated until the
+    present, or channel close time if the channel is closed. Requests with
+    start or end time set will also be restricted to channel open/close time
+    because revenue cannot be generated before open or after close.
+*/
+    rpc ChannelRevenue(ChannelRevenueRequest) returns (ChannelRevenueResponse) {
+        option (google.api.http) = {
+            post: "/v1/revenue"
+            body: "*"
+        };
+    };
```

Comment:
Shouldn't this be integrated with the uptime query, make a start with a framework for querying channel fitness? Or at least mark these methods as experimental so they can be changed at any time without going through the deprecation process.

Reply:
I think maybe the OP made a new call rather than stuffing it into `ListChannels` as computing this (as is) is more expensive than obtaining the current uptime values.

Reply:
> I think maybe the OP made a new call rather than stuffing it into `ListChannels` as computing this (as is) is more expensive than obtaining the current uptime values.

This was the original thought, not to overload listchannels with this call. 
Depending on how we go with exposing revenue info, I'll either add a call which will return uptime and revenue over a start-end period for a specific chan, or go with a `RevenueRepot` which does some pairwise calcs over a range (and leave uptime in `listchannels`).

Reply:
Personally I like seeing the uptime in `listchannels`

Reply:
I am ok with keeping it then. It is marked experimental anyway.